openapi-v2.yaml

```diff
@@ -1968,79 +2010,20 @@ definitions:
       type:
         $ref: "#/definitions/GroupMemberType"
 
-  GroupContentActivity:
-    description: Object containing activity of an asset of a group
-    type: object
-    properties:
-      asset:
-        description: The asset details
-        type: object
-        properties:
-          id:
-            description: The asset ID
-            type: string
-          name:
-            description: The asset name
-            type: string
-          namespace:
-            description: The namespace that the asset belongs to
-            type: string
-          asset_type:
-            description: The type of the asset
-            $ref: "#/definitions/AssetType"
-      activity_log:
-        description: Object containing the activity log
-        type: object
-        $ref: "#/definitions/ArrayActivityLog"
-
-  GroupContentActivityResponse:
-    description: Object containing activity logs of group content along with the pagination metadata
-    type: object
-    properties:
-      activity:
-        description: Activity of a group's content
-        type: array
-        x-omitempty: false
-        items:
-          $ref: "#/definitions/GroupContentActivity"
-      pagination_metadata:
-        x-omitempty: false
-        $ref: "#/definitions/PaginationMetadata"
-
-  GroupActivityLogResponse:
-    description: Object containing activity logs of a group along with the pagination metadata
+  GroupTotalActivityLogResponse:
```

Comment:
Shall we also rename this to GroupActivityResponse to better match its route name?